crates/reliable-broadcast/src/lib.rs

```diff
@@ -10,39 +10,40 @@ use std::{collections::HashMap, sync::Arc, time::Duration};
 pub trait RBMessage: Send + Sync + Clone {}
 
 #[async_trait]
-pub trait RBNetworkSender<M: RBMessage>: Send + Sync {
+pub trait RBNetworkSender<M: RBMessage, R: RBMessage = M>: Send + Sync {
```

Comment:
I don't understand this annotation of R: RBMessage = M, it's probably more natural to call it Req(uest), Res(ponse)